src/components/layout/nav/util/useAuth.tsx

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from 'react'
+import { isLoggedIn } from 'common/utility'
+
+const useAuth = () => {
+    const [is_logged_in, setLoggedIn] = useState(false)
+
+    useEffect(() => {
+        setLoggedIn(isLoggedIn())
+        const checkCookieChange = setInterval(() => setLoggedIn(isLoggedIn()), 800)
+        return () => clearInterval(checkCookieChange)
+    }, [])
+
+    return [is_logged_in]
+}
+
+export default useAuth
```

Comment:
I think this is general enough to be together in the `hooks` folder. Please name it in snake_case for hooks, `use-auth.tsx`.

Reply:
sure @sean-binary I'll move this there